pandas/core/arrays/base.py

```diff
@@ -354,7 +354,8 @@ def _from_sequence_of_strings(
 
         Examples
         --------
-        >>> pd.arrays.IntegerArray._from_sequence_of_strings(["1", "2", "3"])
+        >>> pd.arrays.IntegerArray._from_sequence_of_strings(
+        ...    ["1", "2", "3"], dtype=pd.Int64Dtype())
```

Comment:
```suggestion
        >>> pd.arrays.IntegerArray._from_sequence_of_strings(
        ...     ["1", "2", "3"], dtype=pd.Int64Dtype()
        ... )
```

I _think_ this should fix it?